new_platforms/libsocket/enc/socket_insecure_enc.c

```diff
@@ -457,7 +456,7 @@ oe_getaddrinfo_OE_NETWORK_INSECURE(
                 break;
             }
             strncpy(ai->ai_canonname, response[i].ai_canonname,
-                sizeof(response[i].ai_canonname));
+                sizeof(ai->ai_canonname));
```

Comment:
BUG: this change is wrong since sizeof() here will be sizeof pointer (see line 452 above where it is assigned). Whereas the original code will be size 256.  Please revert!

Reply:
Reverted.